src/Compilers/Core/Portable/InternalUtilities/RoslynString.cs

```diff
@@ -11,5 +11,11 @@ internal static class RoslynString
         /// <inheritdoc cref="string.IsNullOrEmpty(string)"/>
         public static bool IsNullOrEmpty([NotNullWhen(returnValue: false)] string? value)
             => string.IsNullOrEmpty(value);
+
+#if !NET20
```

Comment:
When is this not true?

Reply:
➡️ In the **ResultProvider.NetFX20** project.